core/src/test/scripts/unit_tests_validate_data.py

```diff
@@ -128,7 +128,8 @@ def validate(self, data_filename, validator_class,
                                     self.logger, relaxed_mode,
                                     strict_maf_checks)
         validator.validate()
-        return self.get_log_records()
```

Comment:
I guess this change can be reverted for simplicity

Reply:
Done.